frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -434,6 +435,18 @@ export const dashboardLogic = kea<dashboardLogicType<DashboardLogicProps>>({
                 }
             },
         ],
+        breadcrumbs: [
+            (s) => [s.allItems],
+            (allItems): Breadcrumb[] => [
+                {
+                    name: 'Dashboards',
+                    path: urls.dashboards(),
+                },
+                {
+                    name: allItems?.id ? allItems.name || 'Untitled Dashboard' : null,
```

Comment:
The empty name states are a bit disjointed:

- for dashboards it's "Untitled Dashboard"
- for insights it's "New Insight"
- for actions it's "Action"
- for feature flags it's "New feature flag"

Could we unify these? Maybe just using "Unnamed" everywhere would be enough, since we know what type of entity this is anyway from the previous breadcrumb (e.g. Dashboards › Unnamed)

Reply:
Sure, unified to `Unnamed`